test-suite/squidconf/time_units

```diff
@@ -54,4 +54,3 @@ url_rewrite_timeout 292 year on_timeout=bypass
 url_rewrite_timeout 1 on_timeout=bypass
 
 # TODO: a nanosecond-precison parameter (time-units-small)
-
```

Comment:
If you have to modify this file for another reason, please undo this whitespace change. Otherwise, I can do it for you during the next review round.